src/app/content/actions.ts

```diff
@@ -32,3 +32,5 @@ export const receivePageNotFoundId = createStandardAction('Content/receivePageNo
 
 export const openNudgeStudyTools = createStandardAction('Content/openNudgeStudyTools')();
 export const closeNudgeStudyTools = createStandardAction('Content/closeNudgeStudyTools')();
+
+export const setTextSize = createStandardAction('Content/setTextSize')<number>();
```

Comment:
you should type this and `textResizerValueMap` with a union of const values, or replace `textResizerValueMap` with some kind of math function, otherwise you're setting yourself up to have invalid numbers go in the top and `undefined` printed in the styles

Reply:
Added better typing here with `as const`, does require a few `as TextResizerValue` because of the narrowing when using `.includes` but overall seems to work nicely

Reply:
Ended up going with some minor duplication to explicitly type the values https://github.com/openstax/rex-web/pull/1598/files#diff-51c9e59c9ff2ef74713ad6f6f68e468bc12c90fd9778beab1fa3f710c2733553R24 but maybe there is a better way, I thought a union approach wouldn't narrow it enough? Let me know if you have a suggestion!